test/unit/controllers/payment_reference/post_product_reference_controller_test.js

```diff
@@ -94,7 +94,7 @@ describe('product reference post controller', function () {
     })
   })
 
-  describe('when reference field exceeds max length 255', function () {
+  describe('when reference field exceeds max length', function () {
```

Comment:
Could you update the actual tested value to be just over the limit (i.e. 51 characters)?